js/components/urlBarSuggestions.js

```diff
@@ -0,0 +1,459 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
```

Comment:
Sorry if this is a little nit-picky, but could you put this new file into the new directory structure? EX: `app/renderer/components/`